.github/CODEOWNERS

```diff
@@ -18,6 +18,7 @@ dagcircuit/           @ajavadia @kdk @1ucian0 @maddy-tod
 providers/            @ajavdia @mtreinish @diego-plan9
 basicaer/             @chriseclectic @ajavadia
 pulse/                @lcapelluto @taalexander @eggerdj @nkanazawa1989
+scheduler/            @lcapelluto @taalexander @eggerdj @nkanazawa1989
```

Comment:
Codeowners doesn't mean anything for users who do not have write permission on the repo already, see: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners but since this is already the codeowners list for `pulse/` this is fine. But we should probably change this up at some point to avoid confusion.